openforcefield/topology/topology.py

```diff
@@ -1399,6 +1405,38 @@ def box_vectors(self, box_vectors):
             assert len(box_vectors) == 3
         self._box_vectors = box_vectors
 
+    @property
+    def is_periodic(self):
+        """Return whether or not this Topology is intended to be described with periodic
+        boundary conditions."""
+        return self.box_vectors is not None
+
+    @is_periodic.setter
+    def is_periodic(self, is_periodic):
+        """
+        Set the partial charge model used for all molecules in the topology.
+
+        Parameters
+        ----------
+        is_periodic : bool
+            Whether or not this Topology is periodici
+
+        """
+        if is_periodic is self.is_periodic:
+            warnings.warn(
+                f".is_periodicic is already {is_periodic}. " "Setting has no effect."
+            )
```

Comment:
(not blocking) I could take or leave the "Everything is OK" warning. Your call. 
```suggestion

```

Reply:
Agreed